metpy/plots/_mpl.py

```diff
@@ -328,7 +328,7 @@ def draw(self, renderer):
                     y = y + posy
                     if renderer.flipy():
                         y = canvash - y
-                    clean_line, ismath = self.is_math_text(line)
+                    clean_line, ismath = self._preprocess_math(line)
```

Comment:
So the problem is that `_preprocess_math` is only available on matplotlib 3.1, and we need to support back to matplotlib 2.x currently because of Python 2.7 (and probably would support back to matplotlib 3.0 anyway). That, unfortunately means doing something like:
```python
check_line = getattr(self, '_preprocess_math', self.is_math_text)
clean_line, ismath = check_line(line)
```